docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -237,6 +237,7 @@ truck,something3,buck
 |`valueColumn`|The name of the column containing the value|no|The second column|
 |`delimiter`|The delimiter in the file|no|tab (`\t`)|
 |`listDelimiter`|The list delimiter in the file|no| (`\u0001`)|
+|`multiValueDelimiter`|The list delimiter in the file|no| `null`|
```

Comment:
Description is not useful, it's exactly the same as of `listDelimiter`. And explain more what does "no" (null) mean

Reply:
Thanks, The description should be "The multiValue delimiter in the file", and "no"(null) means that the `multiValueDelimiter` is not required and default value is null, because of empty and null has different meaning for this key.